static/app/utils/replays/hooks/useCurrentItemScroller.tsx

```diff
@@ -3,11 +3,11 @@ import {useEffect, useState} from 'react';
 
 const defer = (fn: () => void) => setTimeout(fn, 0);
 
-export function useCurrentItemScroller(containerRef: RefObject<HTMLElement>) {
+export function useCurrentItemScroller(containerRef: RefObject<HTMLElement> | undefined) {
```

Comment:
```suggestion
export function useCurrentItemScroller(containerRef: RefObject<HTMLElement>) {
```

It's always a ref, never undefined.

`ref.current` might be undefined or null, but there's always a `.current` field.